streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamAggregate.java

```diff
@@ -56,12 +59,15 @@ public void enableSendingOldValues() {
         private KeyValueStore<K, T> store;
         private StreamsMetricsImpl metrics;
         private TupleForwarder<K, T> tupleForwarder;
+        private Sensor skippedRecordsSensor;
 
         @SuppressWarnings("unchecked")
         @Override
         public void init(final ProcessorContext context) {
             super.init(context);
             metrics = (StreamsMetricsImpl) context.metrics();
+            skippedRecordsSensor = skipRecordSensor(metrics);
```

Comment:
Retrieve sensor to record skipped records by using `StreamsMetricsImpl` and the respective static method in `ThreadMetrics`.